corehq/apps/reports/standard/users/reports.py

```diff
@@ -150,12 +149,17 @@ def _for_domains(self):
 
     @property
     def rows(self):
-        records = self._get_queryset().order_by('-changed_at')[
+        records = self._get_queryset().order_by(self.ordering)[
             self.pagination.start:self.pagination.start + self.pagination.count
         ]
         for record in records:
             yield self._user_history_row(record, self.domain, self.timezone)
 
+    @property
+    def ordering(self):
+        by, direction = list(self.get_sorting_block()[0].items())[0]
```

Comment:
since you're not using elasticsearch or multiple sort columns, you might as well just get the column from `request.GET.get(iSortCol_0)` directly

Reply:
Thanks, does that mean it would make sense to do it this way in the future if we wanted to sort by other columns? I ask because I was anticipating that sort functionality would be requested for other columns. I had actually added sorting to other columns to test, but thought it would be better to get the specific request (sorting by timestamp) out and follow up later about other columns.

Reply:
I've tried out some different things to get a better understanding of how sorting is being implemented. If we are only sorting by one column and write `ordering` like below `PaginatedReportMixin` can go away, but a lot of the same work has to be done to get the correct properties and pass them on to `rows` for sorting . I could be missing a simpler approach. Maybe `by` could alway be set as 'changed_at' in this context, and just get `direction` from the request? I'm inclined to go ahead and have sorting available for additional columns like `Via` and `Action`.
```
@property
  def ordering(self):
      col_id = int(self.request.GET.get('iSortCol_0'))
      by = self.headers.header[col_id].prop_name
      if by == None:
        return '-changed_at'
      direction = self.request.GET['sSortDir_0']
      return '-' + by if direction == 'desc' else by
```

Reply:
Good point, if you're supporting sorting by multiple columns it probably does make sense.  `get_sorting_block` is intended as a util to turn the sort parameters into a valid elasticsearch sorting block, which is why the format is a little strange.  I was mostly thinking that you needn't feel obligated to use that if it's weird to work around.  Your conclusions seem reasonable though, as I guess that ES sort syntax is easier to parse than the request params.